packages/sw-runtime-caching/src/lib/stale-while-revalidate.js

```diff
@@ -39,6 +45,23 @@ import assert from '../../../../lib/assert';
  */
 class StaleWhileRevalidate extends Handler {
   /**
+   * Constructor for a new StaleWhileRevalidate instance.
+   *
+   * @param {Object} input
+   * @param {RequestWrapper} [input.requestWrapper] An optional `RequestWrapper`
+   *        that is used to configure the cache name and request plugins. If
+   *        not provided, a new `RequestWrapper` using the
+   *        [default cache name](#defaultCacheName) will be used.
+   */
+  constructor(input = {}) {
+    super(input);
+
+    const cacheableResponse = new CacheableResponse({statuses: [0, 200]});
+    this.requestWrapper.cacheableResponseCheck =
```

Comment:
Again this feels really weird and bad and likely to trip someone up sooner or later.